src/web/model_api.nit

```diff
@@ -120,6 +120,60 @@ class APIEntity
 	end
 end
 
+# Linearize super definitions of a MClassDef or a MPropDef if any.
+#
+# Example: `GET /entity/core::Array/linearization`
+class APIEntityLinearization
+	super APIHandler
+
+	redef fun get(req, res) do
+		var mentity = mentity_from_uri(req, res)
+		if mentity == null then return
+		var arr = new JsonArray
+		if mentity isa MClass then
+			arr.add_all mclass_lin(mentity)
+		else if mentity isa MClassDef then
+			arr.add_all mclassdef_lin(mentity)
+		else if mentity isa MProperty then
+			arr.add_all mproperty_lin(mentity)
+		else if mentity isa MPropDef then
+			arr.add_all mpropdef_lin(mentity)
```

Comment:
~~~
var lin = mentity.lin_for_doc
if lin == null then
  res = 404 (Do not know how to linearize such an entity)
  return
end
~~~